.clang-format

```diff
@@ -0,0 +1,8 @@
+BasedOnStyle:  LLVM
+---
+Language:        Cpp
+AlwaysBreakTemplateDeclarations: true
+BreakBeforeBraces: Custom
```

Comment:
Would this get changed to true to allow for the break I mentioned below?

Reply:
`true` don't work, but `Allman` does.